packages/galpy/meta.yaml

```diff
@@ -1,22 +1,21 @@
 package:
   name: galpy
-  version: 1.8.0
+  version: 1.8.1
   top-level:
     - galpy
 source:
-  url: https://files.pythonhosted.org/packages/73/75/632c028d3cdd948a4ea1edbfbfa568edfd146e03db0ea56d2bacd0aac231/galpy-1.8.0.tar.gz
-  sha256: a69f4a98a9a094ee5eda8acb2f5c7b3a786388d089423a214ab3b290fceddb53
-  patches:
-    - patches/0001-Fix-function-pointer-disagreement.patch
+  url: https://files.pythonhosted.org/packages/97/5b/ef489c571595f753fd86634e93f941897bf3511317ec829f5b080a390bb1/galpy-1.8.1.tar.gz
+  sha256: 9e2b55e50aab4aa7f4313ab9c333ac4512a4338e8d111fce5f6c14456ea53a07
 build:
   script:
-    export LIBGSL_INCLUDE_PATH=$(pkg-config --cflags-only-I --dont-define-prefix gsl)
-    export LIBGSL_LIBRARY_PATH=$(pkg-config --libs-only-L --dont-define-prefix gsl)
+    export LIBGSL_INCLUDE_PATH=$(pkg-config --cflags-only-I --dont-define-prefix
+    gsl) export LIBGSL_LIBRARY_PATH=$(pkg-config --libs-only-L --dont-define-prefix
```

Comment:
It looks weird although probably technically correct. No change needed.

Reply:
Well why not let's put it back... Prettier does seem to like it this way.